### ISSUE TITLE
Automate refreshing data or refresh button!

### ISSUE THREAD

**Issue by khaledkhj** [amitshekhariitbhu/Android-Debug-Database#81]
Thank you for this great work,

It would be really good if you can keep the data or database's tables refreshing by it self 
OR
if we can have like refresh button on each table and each SharedPreferences to refresh data!

Thank you in advance.

**Comment by amitshekhariitbhu** [comment]
Yes, this will be a good feature to add in the new release. Thanks for the suggestion.

Thanks
Amit Shekhar

**Comment by Tsuki** [comment]
Have this function planed on roadmap?

**Comment by amitshekhariitbhu** [comment]
Yes, this it will be present in 1 or 2 releases.

**Comment by erlangp** [comment]
+1
